examples/access_level_vpc_ip/main.tf

```
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2019 Google LLC
```

```suggestion
 * Copyright 2024 Google LLC
```

Here and throughout.